packages/wallet-link/src/walletLink.ts

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable object-curly-newline */
+import Address from '@helium/address'
+import { utils } from '@helium/crypto'
```

Comment:
@tyler-whitman how does this get swapped for `@helium/crypto-react-native` on react native? I'm getting an error when using it in the iOS simulator
![Simulator Screen Shot - iPhone 13 - 2022-07-01 at 08 38 22](https://user-images.githubusercontent.com/1760821/176906038-104e0aad-4245-49bd-8dfd-015657e7acc5.png)

Reply:
yeah you can't import either without causing an error on the wrong platform

Reply:
maybe look at how `@helium/transactions` does its tests? you could probably make a mock signable keypair object for tests. you could also try moving crypto in dev dependencies but that may still get imported if the RN app is being run in dev mode